automation/docker-compose.yml

```diff
@@ -0,0 +1,567 @@
+version: '2.1'
+services:
+
+    zookeeper:
+        image: zookeeper:3.4.9
+        hostname: zookeeper
+        container_name: zookeeper
+        ports:
+            - "2181:2181"
+        environment:
+            ZOO_MY_ID: 1
+            ZOO_PORT: 2181
+            ZOO_SERVERS: server.1=zoo1:2888:3888
+        volumes:
+            - ./zk-single-kafka-single/zoo1/data:/data
+            - ./zk-single-kafka-single/zoo1/datalog:/datalog
+
+    kafka:
+        image: confluentinc/cp-kafka:3.2.0
+        hostname: kafka
+        ports:
+            - "9092:9092"
+        environment:
+            KAFKA_ADVERTISED_LISTENERS: "PLAINTEXT://kafka:9092"
+            KAFKA_ZOOKEEPER_CONNECT: "zookeeper:2181"
+            KAFKA_BROKER_ID: 1
+            KAFKA_LOG4J_LOGGERS: "kafka.controller=INFO,kafka.producer.async.DefaultEventHandler=INFO,state.change.logger=INFO"
+            KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+        volumes:
+            - ./zk-single-kafka-single/kafka1/data:/var/lib/kafka/data
+        depends_on:
+            - zookeeper
+        
+    mongo:
+        image: 'mongo:latest'
+        container_name: 'mongo'
+        ports:
+            - '27100:27017'
+    
+    neo4j:
+        environment:
+            - NEO4J_AUTH=none
+        image: 'neo4j:latest'
+        container_name: 'neo4j'
+        ports:
+            - '7688:7687'
+
+    vault:
+        image: 'vault:0.9.5'
+        ports:
+            - "127.0.0.1:8200:8200"
+        container_name: 'vault'
+    
+    elasticsearch:
+        image: 'elasticsearch:latest'
+        ports:
+            - '9201:9200'
+        container_name: 'elasticsearch'
+
+    auth_api:
+        container_name: 'zebedee'
```

Comment:
could it get confusing if this is called zebedee if someone expects it to be auth_api? For example, when looking at the logs